helm-charts/fluentd-es/templates/fluentd-es.yaml

```diff
@@ -56,6 +56,31 @@ roleRef:
   name: {{  template "fluentd-es.fullname" . }}
   apiGroup: ""
 ---
+apiVersion: v1
+kind: Service
+metadata:
+  name: fluentd-es-metrics
+  namespace: logging
```

Comment:
Please omit `namespace` as with the other resources. `helm` manages it separately.